protostar-rust/src/cheatcodes_hint_processor.rs

```diff
@@ -202,32 +224,68 @@ fn call_contract(
     Ok(return_data)
 }
 
-#[allow(unused, clippy::too_many_lines)]
 fn execute_cheatcode_hint(
     vm: &mut VirtualMachine,
-    exec_scopes: &mut ExecutionScopes,
-    hint: &ProtostarHint,
+    _exec_scopes: &mut ExecutionScopes,
+    blockifier_state: &mut CachedState<DictStateReader>,
+    selector: &BigIntAsHex,
+    [input_start, input_end]: [&ResOperand; 2],
+    [output_start, output_end]: [&CellRef; 2],
```

Comment:
why do you use slices here? You can just receive 4 args instead.

Reply:
I guess the only argument for using slices would be that clippy complains about too many arguments. But it's not convincing me, changing to use normal arguments.

Reply:
Slices here are far less readable to me :/